content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -0,0 +1,464 @@
+```
+
+## Adding Tina
+
+Awesome! You're all set up and ready start adding TinaCMS. You can initialize it using the command below.
+
+```powershell
+npx @tinacms/cli@latest init
+```
+
+After running the command above you'll receive a few propmts
+
+1. When prompted to select a framework select `“other”`
+2. Choose `“NPM”` as your package manager
+3. When asked if you'd like to use Typescript choose `“no”`
+4. Set the public assets location to `“public”`
+
+## Setting up Gatsby for Tina
+
+Now that we've added Tina to our project there's a few more steps we need to do to get Tina working with Gatsby. Add the following line at the top of `tina/config.js`
+
+```diff
+export default defineConfig({
++  client: { skip: true },
+// ...
+```
+
+Then add the following line in `gatsby-node.js`
+
+```diff
++ const express = require("express");
++ exports.onCreateDevServer = ({ app }) => {
++   app.use("/admin", express.static("public/admin"));
++ };
+```
+
+To save ourself some pain we can also change our startup command to make sure Tina gets run when we run our app in development mode.
+
+```diff
+  "scripts": {
+    "build": "gatsby build",
+-   "develop": "gatsby develop",
++   "develop": "npx tinacms dev -c \"gatsby develop\"",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "start": "gatsby develop",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+  }
+```
+
+## Configuring our Schema
+
+Now we've added Tina to our project but it's not pointing at our existing markdown files **(Oh noes!)**. Let fix that. Open up `tina/config.ts` and change the path to point to our blog directory. We'll also need to update our schema since we'll be working with mdx files.
+
+```diff
+schema: {
+  collections: [
+    {
+      name: "post",
+      label: "Posts",
++     format: "mdx",
+-     path: "content/posts"
++     path: content/blog"
+// ...
+```
+
+We'll also need to change the location where images get uploaded to. Since `"public"` doesn't get tracked in Git.
+
+> By moving our images to \`"static"\`, we're ensuring that they'll be tracked in git and bundled at run time.
+
+```diff
+export default defineConfig({
+  branch,
+
+  client: { skip: true },
+
+  // Get this from tina.io
+  clientId: process.env.NEXT_PUBLIC_TINA_CLIENT_ID,
+  // Get this from tina.io
+  token: process.env.TINA_TOKEN,
+
+  build: {
+    outputFolder: "admin",
+    publicFolder: "public",
+  },
+  media: {
+    tina: {
+-     mediaRoot: "",
++     mediaRoot: "images",
+-     publicFolder: "public",
++     publicFolder: "static",
+
+    },
+// ...
+```
+
+## Setting up TinaMarkdown
+
+Now our project is set up to use TinaCMS. There's a problem though: our project is still using Gatsby's mdx parser instead of TinaCMS's. We'll need to to a bit of tweaking to get it working. First we'll edit the graphql query inside of `gatsby-node.js` so that it returns the raw markdown.
```

Comment:
"to to"?
```suggestion
Now our project is set up to use TinaCMS. There's a problem though: our project is still using Gatsby's mdx parser instead of TinaCMS's. We'll need to do a bit of tweaking to get it working. First we'll edit the graphql query inside of `gatsby-node.js` so that it returns the raw markdown.
```

Reply:
was rephrased to

> With the project set up to use TinaCMS, we still need to switch from Gatsby’s MDX parser to TinaCMS’s. Begin by modifying the GraphQL query in gatsby-node.js to return raw markdown: